tests/axi/test_axi.py

```diff
@@ -296,7 +296,7 @@ def cycle_pause():
     return itertools.cycle([1, 1, 1, 0])
 
 
-if cocotb.SIM_NAME:
+if hasattr(cocotb, 'SIM_NAME') and cocotb.SIM_NAME:
```

Comment:
If this is supposed to be doing a check if the simulation is running use `cocotb.is_simulation`.